### ISSUE TITLE
Separate windows on the first move does a little bit change size 

### ISSUE THREAD

**Issue by chordig** [matkatmusic/ChordieApp_IssueTracker#11]
When I turn on "Switch to separate windows" almost each of the windows on the first move does a little bit change size with such flashing. On the next move it is not doing it.
This is best illustrated by this video:
[https://youtu.be/crArf4-WD4o](url)
My system is Windows 8.1.

**Comment by matkatmusic** [comment]
I'm surprised it even runs on that OS... That OS is not supported by this build.

**Comment by matkatmusic** [comment]
Thanks for sharing the video.  I'll see if I can replicate it here on Windows 10.   If I can't, then I'm not sure what can be done.

**Comment by matkatmusic** [comment]
I am not able to replicate this on Windows 10 with the latest build.

**Comment by chordig** [comment]
@matkatmusic In case of a potential fix for this, I can check on my Windows 8.1 and describe the effect. Generally it works nice on Windows 8.1 :)

**Comment by matkatmusic** [comment]
When I drag the windows on Windows 10, there is a single resizing that I noticed due to the windows being locked to proportional dimensions.   I'm not able to re-create the flashing thing that you saw though.

**Comment by chordig** [comment]
I also tried it on my Windows 10 system in VirtualBox. When I turn on "Switch to separate windows" the bottom edge goes up the first time you move the window.
I am posting a video:
https://youtu.be/3bLZGdwK_1k

As for Windows 8.1 (I did not run this program in Windows 10 before), I will add that it did not happen in the previous version of the program. In the current version, this does not happen with the main application window (before choosing the Switch to separate windows option) and after choosing Switch to separate windows, this does not happen with the "ChordieApp" window.
Other windows, once I move or resize them, don't behave like that anymore.

Although this is probably not exactly about this situation, but maybe something in this topic would be related to this: https://stackoverflow.com/questions/26700236/flicker-when-moving-resizing-window
or
https://stackoverflow.com/questions/20958574/flickering-on-window-when-resizing-from-left-side

If necessary I can help you check something on my Windows 8.1.
